lib/rubygems/gemcutter_utilities.rb

```diff
@@ -250,11 +251,28 @@ def request_with_otp(method, uri, &block)
     end
   end
 
-  def ask_otp
-    say "You have enabled multi-factor authentication. Please enter OTP code."
+  def ask_otp(credentials)
+    webauthn_url = webauthn_verification_url(credentials)
+    if webauthn_url
```

Comment:
Is there any styling problem to merge both lines?

`if webauth_url = ...`

Reply:
@simi  Done! ✅